samcli/lib/iac/cfn/helpers.py

```diff
@@ -0,0 +1,18 @@
+"""
+CFN IaC helper functions
+"""
+from typing import Dict
+
+from samcli.lib.iac.plugins_interfaces import Stack
+from samcli.lib.providers.sam_base_provider import SamBaseProvider
+from samcli.lib.providers.sam_stack_provider import SamLocalStackProvider
+
+
+def get_resolved_cfn_template(
```

Comment:
Why do we need to define an extra function to just wrap `SamLocalStackProvider.merge_parameter_overrides(...)` and do nothing else?